include/cpeabs.h

```diff
@@ -39,7 +39,7 @@
 /**
  * @brief Enables or disables debug logs.
  */
-#if defined DEVICE_GATEWAY && defined BUILD_YOCTO
+#if defined DEVICE_GATEWAY && defined BUILD_YOCTO && defined DEVICE_VIDEO
 
```

Comment:
This AND (&) condition is not correct , here all 3 should be on to enable logging . For broadband devices video flag will not be ON, and it will go to else, this condition should be split and add separate OR for video .

Reply:
Made change:
#if defined DEVICE_GATEWAY && defined BUILD_YOCTO || defined DEVICE_VIDEO